test-case/check-pause-resume.sh

```diff
@@ -39,6 +45,10 @@ test_mode=${OPT_VALUE_lst['m']}
 repeat_count=${OPT_VALUE_lst['c']}
 #TODO: file name salt for capture
 file_name=${OPT_VALUE_lst['f']}
+# configure random value range
+rnd_min=${OPT_VALUE_lst['i']}
+rnd_max=${OPT_VALUE_lst['a']}
+rnd_range=$[ $rnd_max - $rnd_min ]
```

Comment:
Do we need more check here to make sure user input valid value?
EG: check if max >= mix

Reply:
add basic logic is ok for me,  
but for the other hand,  
whether need to the value is string/number for the all parameter

Reply:
Then let add a TODO here.